tofu/imas2tofu/__init__.py

```diff
@@ -11,8 +11,10 @@
 try:
     try:
         from tofu.imas2tofu._core import *
+        from tofu.imas2tofu._mat2ids2calc import *
     except Exception:
         from ._core import *
+        from ._mat2ids2calc import *
```

Comment:
is there a reason why you do an "import *" ? is it just because the file is small?
I'm still approving the PR :)

Reply:
It's because the file is small and also because it contains a very specific use-case that may get generalized later, in which case we'll be happy to have it in a separate module that we can import (becasue it would then get bigger)

Thanks !